modules/network-observability-includelist-example.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-high-traffic-alert_{context}"]
+= Creating alerts
+You can create custom Prometheus rules for the Netobserv dashboard metrics to trigger alerts when some defined conditions are met.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role or with view permissions for all projects.
+* You have the Network Observability Operator installed.
+
+.Procedure
+
+. Create a YAML file by clicking the import icon, *+*.
```

Comment:
@jotak I just want to confirm if its OK to present the example like this? Creating the YAML file in the web console, and the prerequisites are OK?

Reply:
nice, everything looks good!